NVActivityIndicatorView/NVActivityIndicatorView.swift

```diff
@@ -355,10 +355,10 @@ public class NVActivityIndicatorView: UIView {
      
      - returns: The activity indicator view.
      */
-    public init(frame: CGRect, type: NVActivityIndicatorType = DEFAULT_TYPE, color: UIColor = DEFAULT_COLOR, padding: CGFloat = DEFAULT_PADDING) {
```

Comment:
I made this nil-able so I can feed pass-thru nil parameters from other functions but still retain default values. This was opposed to making the default constants public instead of private, which I didn't want to mess up the public API.